pkg/controller/metrics.go

```diff
@@ -0,0 +1,51 @@
+package controller
```

Comment:
This file needs the correct boilerplate.

Reply:
I think you also need to run gofmt over the files.

Reply:
Fixed, thanks!